dockerfiles/Dockerfile.openshift

```diff
@@ -1,6 +1,6 @@
 # node-build stage
 
```

Comment:
These changes needs to be implemented in the file since I did not made this .openshift as I felt it would be removed. 

https://github.com/tl-its-umich-edu/instructor-productivity-tools/pull/348/files

Reply:
After the mysql version upgrade the docker build was failing so I needed to update it as shown in the PR.

Reply:
Okay thanks for the catch.